docs/_includes/favicons.html

```diff
@@ -1 +0,0 @@
-<link rel="icon" type="image/png" href="/favicon.png">
```

Comment:
The current docs do have a favicon included: https://github.com/auth0/auth0-cli/blob/main/docs/_includes/favicon.png. I'm not sure about removing this file tbh 🤔 Or is this coming from the theme itself?

Reply:
This should be configured in the [config](https://github.com/auth0/auth0-cli/pull/615/files#diff-f64371f310bcd3bbf6ead5827d929a4eb13b7516806d6658485a996e39eb2556R17) now but it seems like the live site doesn't have it (although it did work locally at one point). I'll take a look

Reply:
Ah shoot, it used to work before I reverted to rc3 (for a fix for the padding on the search box) as it was added in rc4 - https://github.com/just-the-docs/just-the-docs/blob/main/CHANGELOG.md#new-features.

I'll see if I can add a custom head file that will tide us over til there's a 0.3.0 release or an rc5

Reply:
Resolved in 96a4182315470a94127c2e67d646e50f8e2a9882